ompi/mca/coll/han/coll_han_allreduce.c

```diff
@@ -121,11 +127,11 @@ mca_coll_han_allreduce_intra(const void *sbuf,
     int root_low_rank = 0;
     /* Create t0 task for the first segment */
     mca_coll_task_t *t0 = OBJ_NEW(mca_coll_task_t);
-    /* Setup up t0 task arguments */
+    /* Setup up t0 task argsments */
```

Comment:
Minor: global replacement of `argu` seems to have messed up some of the comments (`argument` to `argsment`)